ocs_ci/utility/csr.py

```diff
@@ -169,3 +173,13 @@ def wait_for_all_nodes_csr_and_approve(timeout=900, sleep=10, expected_node_num=
         pending_csrs = get_pending_csr()
         if pending_csrs:
             approve_csrs(pending_csrs)
+        # In vSphere deployment it sometime happes that VM doesn't get ip and
+        # then we need to restart it to make our CI more stable and let the VM
+        # to get IP and continue with loading ignition config. The rester of
+        # the VMs happens only once in reboot_timeout (120 seconds).
+        if (
+            config.ENV_DATA["platform"] == constants.VSPHERE_PLATFORM
+            and start_time >= reboot_timeout
```

Comment:
`start_time` is seconds since epoch and will always be greater than `reboot_timeout`. Is `time.time()` really the value we want here or did we need another value like `time_since_start`?

Reply:
yep, I forgot to add time.time() - start_time  ;)